mammamiradio/admin.html

```
@@ -648,193 +1013,257 @@
+    <!-- TopBar -->
+    <div class="a-topbar" id="topBar">
+      <span class="live-pill"><span class="dot"></span> <span id="topBarStatus">On Air</span></span>
+      <div class="a-eq" id="topBarEq">
+        <span></span><span></span><span></span><span></span>
+      </div>
+      <div class="meta">
+        <div class="eyebrow">Now on air</div>
+        <div class="np-title" id="topBarTitle">Waiting for signal… <span class="by" id="topBarArtist"></span></div>
+      </div>
+      <div class="a-topbar-tools">
+        <button class="btn-icon" title="Next song" onclick="doSkip(this)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)